README.md

```diff
@@ -271,6 +271,30 @@ package = Package.new(:dimensions => { 'width' => "2.2", :height => 2, "length"
 package.dimensions # => { :width => 2.2, :height => 2.0, :length => 4.5 }
 ```
 
+### Attribute extensions
+
+``` ruby
+module Virtus::Attribute::Format
+  def self.extend?(attribute)
```

Comment:
The API for this isn't terribly clear. Can we add some docs on what this is for?